content/docs/glossary/claimable-balance.mdx

```diff
@@ -0,0 +1,75 @@
+
+## Relevant operations
+
+[Create Claimable Balance](../start/list-of-operations.mdx#create-claimable-balance)
+
+### Parameters
+
+1. Asset - Asset that will be held in the ClaimableBalanceEntry.
+
+2. Amount - Amount of asset stored in the ClaimableBalanceEntry.
+
+3. List of Claimants - A Claimant is an object that holds both the account that can claim the ClaimableBalanceEntry, and a ClaimPredicate that must evaluate to true for the claim to succeed. A ClaimPredicate is a recursive data structure that can be used to construct complex conditionals. Here are some examples -
+   - Can claim at anytime - `CLAIM_PREDICATE_UNCONDITIONAL`
+   - Can claim if the ledger close time in which the claim was included is before X seconds + the ledger close time in which the ClaimableBalanceEntry was created  -  `CLAIM_PREDICATE_BEFORE_RELATIVE_TIME(X)`
+   - Can claim if the ledger close time in which the claim was included is before X (Unix timestamp) - `CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X)`
+   - Can claim if the ledger close time in which the claim was included is after X seconds + the ledger close time in which the ClaimableBalanceEntry was created - `CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_RELATIVE_TIME(X))`
+   - Can claim if the ledger close time in which the claim was included is at or after X (Unix timestamp) - `CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X))`
+   - Can claim between X and Y Unix timestamps (given X < Y) - `CLAIM_PREDICATE_AND(CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X)), CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(Y))`
+   - Can claim outside X and Y Unix timestamps (given X < Y) - `CLAIM_PREDICATE_OR(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(X), CLAIM_PREDICATE_NOT(CLAIM_PREDICATE_BEFORE_ABSOLUTE_TIME(Y))`
+
+### Operation Information
+
+This operation will move Amount of Asset from the operation source account into a new ClaimableBalanceEntry.
+
+Note that the baseReserve requirement for a ClaimableBalanceEntry is dependant on the number of Claimants. The [minimum balance](./minimum-balance.mdx) of the account will increase by `# of Claimants * baseReserve`.
+
+### BalanceID
+
+A successful Create Claimable Balance operation will return a balanceID, which is required to claim the ClaimableBalanceEntry using the Claim Claimable Balance operation. The balanceID is the SHA-256 hash of the OperationID (see [stellar-transactions.x](https://github.com/stellar/stellar-core/blob/master/src/xdr/Stellar-transaction.x)), which contains the transaction source account, sequence number, and the operation index of the CreateClaimableBalance operation in this transaction.
```

Comment:
I just might be going crazy from reading "claim claimable balance" in various forms, but maybe better wording would be

> which is the required parameter when actually claiming the newly-created entry with the [Claim Claimable Balance](link goes here) operation.

to avoid the word "claim" appearing 5x in the sentence :sweat_smile:

Reply:
Also, I think the `balanceID` also includes a discriminant prefixing the hash, so maybe "contains the SHA-256 hash"?

Reply:
Agreed with @Shaptic about being more precise about what a `BalanceID` is. Same question about `OperationID` as `LedgerKey`, if we are going to expose these then maybe they should have documentation pages? We could even have a miscellaneous objects page to avoid too much proliferation.

Reply:
Changed the wording and added a miscellaneous core objects page with an explanation for balanceID.